CONTRIBUTING.md

```diff
@@ -0,0 +1,195 @@
+Either comment on the respective meeting discussion, reach out to us on Matrix (see below), or simply join the meeting and speak up in the meeting's Q&A.
+
+**Flatcar Office Hours are on the second Tuesday of every month at 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST**
+
+* Meeting agendas are published in advance - check our [discussions section](../../discussions/categories/office-hours-agenda) for examples.
+* Call link: [https://meet.flatcar.org/OfficeHours](https://meet.flatcar.org/OfficeHours)
+* A Youtube live stream (which also serves as the meeting's recording) will be published on the respective agenda when a meeting starts.
+
+
+**Flatcar Developer Syncs commence every 4th Tuesday of a month at 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST**
+
+While release planning is a recurring part of each community call we also conduct separate Developer Syncs for backlog grooming and task planning. We discuss Roadmap items, special projects, and day-to-day issues in these calls. If you want to participate and discuss or pick up work, that call is for you!
+Just like the Office Hours the call includes a brief Release Planning with an update on the changes in the next immediate releases.
+
+* Meeting agendas are published in advance - check our [discussions section](../../discussions/categories/flatcar-developer-sync) for examples.
+* Call link: [https://meet.flatcar.org/OfficeHours](https://meet.flatcar.org/OfficeHours)
+* A youtube live stream (which also serves as the meeting's recording) will be published on the respective agenda when a meeting starts.
+
+## Report bugs and request features
+
+Please file a respective [issue](issues) right here in the top-level Flatcar github project.
+For instance, please use the "New Package Request" issue type to [file your request](https://github.com/flatcar/Flatcar/issues/new/choose). Please see [adding new packages to the Flatcar Linux OS image](adding-new-packages.md) for general guidelines.
+
+We have good first issues for new contributors and help wanted issues suitable for any contributor. [good first issue](https://github.com/flatcar/Flatcar/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) has extra information to help you make your first contribution. [help wanted](TODO) are issues suitable for someone who isn't a core maintainer and is good to move onto after
+your first pull request.
+
+Sometimes there won’t be any issues with these labels. That’s ok! There is likely still something for you to work on. If you want to contribute but you don’t know where to start or can't find a suitable issue, you can ⚠️ **explain how people can ask for an issue to work on**.
+
+Once you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it. Something like "I want to work on this" is fine.
+
+## Pull Request Lifecycle
+<todo>
```

Comment:
```suggestion

Pull requests may be issued from repository branches or from a fork of the repo.
Branch PRs are limited to Maintainers since these require write access to the respective repository.
While the origin of a PR may have impact on the approvals required for the CI to build and test, the project does not discriminate PR origins regarding getting a change reviewed or merged.

PRs need to undergo a successful build+test as well as a review, and require at least one LGTM from a maintainer who is not the author of the PR for a merge.
Note that the approver(s) of a PR may be co-authors though - this allows reviewers to suggest changes.

A PR undergoes the following stages:
1. PR filed. <br/>
    This includes filing "draft" PRs which are not considered ready for review and will only undergo build+test  when explicitly requested.
2. PR is ready for review. <br/>
    In this stage, maintainers can start reviewing the PR as well as approve build+test runs.
    PRs can be filed in this stage if the author considers the PR ready at the time of creation.
    To generate traction on stale PRs, authors or project coordinators may actively reach out to the maintainers team, e.g via the Matrix channel or in office hours and developer syncs.
3. PR is being reviewed. <br/>
  Maintainers add comments and may request changes.
  Reviewers may also add change suggestions, streamlining the review process for the author.
  During review, the PR is also vetted against Flatcar's mission statement and our core principles.
  Multiple build + test runs may be approved by maintainers to further validate the PR.
4. PR is being merged, or closed w/o merge. <br/>
  Depending on the outcome of the review, the PR is being merged or closed without a merge.
```